packages/remix-server-runtime/cookies.ts

```diff
@@ -116,7 +116,9 @@ export const createCookieFactory =
       async serialize(value, serializeOptions) {
         return serialize(
           name,
-          value === "" ? "" : await encodeCookieValue(sign, value, secrets),
+          (value as any) === ""
+            ? ""
+            : await encodeCookieValue(sign, value, secrets),
```

Comment:
This can be reverted

```suggestion
          value === "" ? "" : await encodeCookieValue(sign, value, secrets),
```

Reply:
This is not the problem now

Reply:
The problem is within `sessions.ts` now
https://github.com/remix-run/remix/runs/6507483095?check_suite_focus=true#step:6:125